docs/js/features/odata/generate-client.md

```diff
@@ -0,0 +1,115 @@
+---
+id: generator-js-sdk
+title: Generate a typed OData client for JavaScript
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Generate an OData client for JavaScript
+keywords:
+- sap
+- cloud
+- sdk
+- odata
+- JavaScript 
+- TypeScript
+- generate
+author: Charles Dubois
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Generate a typed OData client with the OData Generator
+
+The generator allows you to generate a custom OData client library for any service. You can then easily access these services via the client libraries.
+
+There are two way to use the generator:
+- Via the command line interface (CLI)
+- By instantiating and invoking it at runtime
```

Comment:
I wouldn't make a list here, this is not important enough to highlight it that way. Maybe just: "The SAP Cloud SDK generator can both be used as a command line interface (CLI) and programmatically."